modules/migration-state-migration-cli.adoc

```diff
@@ -0,0 +1,126 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/about-mtc-3-4.adoc
+// * migration_toolkit_for_containers/about-mtc.adoc
+
+[id="migration-state-migration-cli_{context}"]
+= Migrating an application's state
+
+You can use {mtc-full} ({mtc-short}) to migrate an application's state. {mtc-short} copies persistent volume claims (PVCs) and specific Kubernetes objects, for example, `Secret` custom resources (CRs) or config maps that store an application's state.
```

Comment:
please refer to this description 

> State migration
During a state migration, PV data is copied to the target cluster. PV references are not moved, and source pods continue running. Select the persistent volumes you want to migrate and optionally rename the PVC for each on the target cluster.

Reply:
@XinRedhat  Where did you get this information? Can you send me a link?

Reply:
I will add the part about renaming the PVC to the subsection on migrating PVCs.